gulp/tasks/test.js

```diff
@@ -283,7 +282,8 @@ function spawnProtractor( ) {
     }
   }
 
-  if ( gulpUtil.env.sauce === 'true' ) {
+  const commandLineParams = minimist( process.argv.slice( 2 ) ) || {};
+  if ( commandLineParams.sauce === 'true' ) {
```

Comment:
This is going to be different behavior, isn't it? Before it was an environment variable; after it's a command-line flag, or am I reading that wrong?

Reply:
no, it was always a command line flag. We are going to do the same thing for headless. You still need to have your sauce credentials set though.

Reply:
Ah okay, thanks!